invenio/base/static/js/settings.js

```diff
@@ -53,6 +53,8 @@ require.config({
     "jasmine-events": "js/jasmine/events_checker",
     "jasmine-initialization": "js/jasmine/initialization_checker",
     "select2": "vendor/select2/select2.min",
+    "ckeditor-core": "vendors/ckeditor/ckeditor",
+    "ckeditor-jquery": "vendors/ckeditor/adapters/jquery"
```

Comment:
A question to this. According to one of RFCs we will probably go with Airbnb JS Style Guid and there is something like:
> Additional trailing comma: Nope. This can cause problems with IE6/7 and IE9 if it's in quirksmode. Also, in some implementations of ES3 would add length to an array if it had an additional trailing comma.
([source](https://github.com/airbnb/javascript#commas))

`settings.js` probably won't cause any problems mentioned in aboved quote anyway. 
A question arises here to me: Will we verify all `js` files against this style guide or will we use it only to some front-end components?